packages/server/config/config.default.js

```diff
@@ -82,6 +82,27 @@ module.exports = {
      * Append to existing pipelines
      */
     // config.pipelines.push(myPipelines[0]);
+
+
+    /* Integration Plugins */
+
+    // Paperless Integration example
+    //
+    // Prevent accessing secrets from the UI
+    // config.plugin_secrets = {
+    //   paperless: {
+    //     token: 'xxxxxxxxxx'
```

Comment:
A link or a short description, of how to obtain the token might be helpful (it is a rather unknown feature of paperless).